src/WebJobs.Script/Diagnostics/HealthChecks/HealthCheckExtensions.cs

```diff
@@ -126,6 +128,24 @@ public static IHealthChecksBuilder AddScriptHostHealthCheck(this IHealthChecksBu
             return builder;
         }
 
+        /// <summary>
+        /// Adds a health check for the WebJobs storage account.
+        /// </summary>
+        /// <param name="builder">The builder to register health checks with.</param>
+        /// <returns>The original builder, for call chaining.</returns>
+        public static IHealthChecksBuilder AddWebJobsStorageHealthCheck(this IHealthChecksBuilder builder)
+        {
+            ArgumentNullException.ThrowIfNull(builder);
+
+            // Ensure singleton as this health check refreshes in the background.
+            builder.Services.TryAddSingleton<WebJobsStorageHealthCheck>();
+            builder.AddCheck<WebJobsStorageHealthCheck>(
+                HealthCheckNames.WebJobsStorage,
+                tags: [HealthCheckTags.Configuration, HealthCheckTags.WebJobsStorage],
```

Comment:
Isn't this more connectivity than config?

Reply:
Good point. Why not both?